src/KPM.jl

```diff
@@ -106,10 +110,10 @@ function addmomentaKPM!(b::KPMBuilder{<:AbstractMatrix,<:AbstractSparseMatrix},
     fill!(ket0, zero(eltype(ket0)))
     mul!(ket1, A´, ket)
     μlist[1] += proj(ket1, ket)
-    ProgressMeter.next!(pmeter; showvalues = ())
+    #ProgressMeter.next!(pmeter; showvalues = ())
```

Comment:
Do remove the comments if you don't want the code

Can you explain why ProgressMeter interferes with this? I seem to recall that ProgressMeter can play nicely with threads now, it might be worth a look

Reply:
Yes, there is no conflict with threads. However, progressmeter slowed a bit the calculations. This is why I commented it. Let me go over it again I guess it is likely that this extra time is negligible for large systems.

Reply:
Maybe useful: https://discourse.julialang.org/t/progressmeter-and-threads/11537

It seems to suggest there is a proper way to do things without locking the threads (didn't read it carefully)

Reply:
Thanks! I'll explore and implement it. This looks like the correct way to do it.